<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarWidget</class>
 <widget class="QWidget" name="CalendarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>586</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calendar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/Graphics/rpgassistant.ico</normaloff>:/Graphics/rpgassistant.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layoutDate">
     <item>
      <widget class="QSpinBox" name="spinDay">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboMonth"/>
     </item>
     <item>
      <widget class="QSpinBox" name="spinYear">
       <property name="minimum">
        <number>-10000</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboReckoning"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableCalendar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>205</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutNavigation">
     <item>
      <widget class="QPushButton" name="buttonPreviousMonth">
       <property name="text">
        <string>&amp;&lt; Month</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPreviousDay">
       <property name="text">
        <string>&amp;- Day</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNextDay">
       <property name="text">
        <string>Day &amp;+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNextMonth">
       <property name="text">
        <string>Month &amp;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="layoutMoonPhase">
     <item row="0" column="1" rowspan="2">
      <widget class="QSvgWidget" name="widgetMoonPhase" native="true">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelMoonPhase">
       <property name="text">
        <string>Moon phase:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelMoonPhaseText">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelHolidays">
     <property name="text">
      <string>Holidays:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textHolidays"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header location="global">QSvgWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
